packages/editor/src/client/blocks/linkPageBreakBlock.tsx

```diff
@@ -11,44 +32,174 @@ export function LinkPageBreakBlock({
   const focusRef = useRef<HTMLTextAreaElement>(null)
+  const focusInputRef = useRef<HTMLInputElement>(null)
 
   useEffect(() => {
     if (autofocus) focusRef.current?.focus()
   }, [])
 
+  const [isChooseModalOpen, setChooseModalOpen] = useState(false)
+  const [isEditModalOpen, setEditModalOpen] = useState(false)
+
   return (
     <>
+      <Box flexGrow={1} style={{marginBottom: '20px'}}>
+        <TextInput
+          ref={focusInputRef}
+          placeholder="Title"
+          label="Title"
+          style={{fontSize: '24px'}}
+          value={text}
+          disabled={disabled}
+          onChange={e => onChange({...value, text: e.target.value})}
+        />
+      </Box>
       <Box flexGrow={1}>
         <TypographicTextArea
           ref={focusRef}
-          variant="h2"
-          placeholder="Text"
-          align="center"
-          value={text}
+          variant="body1"
+          placeholder="Optional Text"
+          align="left"
+          value={htmlText}
+          onChange={e => onChange({...value, htmlText: e.target.value})}
+        />
+      </Box>
+      <Box style={{width: '50%', display: 'inline-block'}}>
+        <TextInput
+          ref={focusInputRef}
+          placeholder="Link Text"
+          label="Button label"
+          value={linkText}
           disabled={disabled}
-          onChange={e => onChange({...value, text: e.target.value})}
+          onChange={e => onChange({...value, linkText: e.target.value})}
+        />
+      </Box>
+      <Box style={{width: '50%', display: 'inline-block', padding: '10px'}}>
+        <TextInput
+          ref={focusInputRef}
+          placeholder="Link URL"
+          label="Button link"
+          value={linkURL}
+          disabled={disabled}
+          onChange={e => onChange({...value, linkURL: e.target.value})}
         />
       </Box>
-      <TypographicTextArea
-        ref={focusRef}
-        variant="body1"
-        placeholder="Link Text"
-        align="center"
-        value={linkText}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkText: e.target.value})}
-      />
-      <TypographicTextArea
-        ref={focusRef}
-        variant="subtitle2"
-        placeholder="Link URL"
-        align="center"
-        value={linkURL}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkURL: e.target.value})}
-      />
+      <br />
+      <br />
+      <div className={'option-wrapper'} style={{display: 'flex'}}>
+        <Card
+          overflow="hidden"
+          width={200}
+          height={150}
+          flexGrow={1}
+          marginRight={Spacing.ExtraSmall}>
+          <Box padding={'10'}>
+            <p>Colors and Styles</p>
+            <small>Styles: </small>
+            <select
+              defaultValue={styleOption}
+              onChange={e => onChange({...value, styleOption: e.target.value || ''})}>
+              <option value="default">Default Color</option>
```

Comment:
Not happy with that. How should other we.publish instance user expand this.

One option is to add a settings page in the editor where you can add your own Styles with label, Name and description or you change this field to a textinput and just pass a css class name. As i mentioned in the stand up we'll implement a "extend with styles" solution for all blocks soon but having fixed values in the editor is not a suitable option.

Reply:
MVP Approach, this is the suitable way for now as, this was cleary defined in offer before starting. (MVP Approach)
.we need a release of api to start local development on Bajout FE side ... (rather considering to make a `next` release wich is release at least every day (and fix the meniotoned stuff by you before new versioned is published release)

Reply:
MPV, ok for now, can be solved on clean up PR after FE Bajour implementation is done